components/GymPlan/GymPlan.js

```diff
@@ -1,8 +1,13 @@
 import WorkoutSet from "./WorkoutSet";
 
 export default function GymPlan() {
+
+  function handleSubmit(event) {
+    event.preventDefault();
+  }
```

Comment:
Where does your data go? Right now it stays in the input fields, right? I guess I would add some kind of "card" with the input that you submit so that the data is stored in a div or something like that and not stay in the input fields.

Reply:
Currently the handleSubmit function is only there, because there is a form tag around the WorkoutSets. 
It was a normal wrapper before, but because there are input fields in each WorkoutSet, I think they should be connected with a form, but I'm not 100% sure. 

Since I changed it to form, I had to build up a function handleSubmit, so the set, if it's added  with the button "add exercise" it doesn't get lost & the button won't be mixed up with the submit button (because, I guess, the form tag gets automatically a relation with the button as the "submit" button. )